src/vpn-worker.ts

```diff
@@ -127,8 +127,6 @@ const worker = async (instanceId: number, serviceId: number) => {
 					const cn = client.uuid;
 					try {
 						logger.info(`disconnecting ${cn}`);
-						// update device state
-						clients.setConnected(cn, serviceId, false, logger);
```

Comment:
I removed this because disconnecting the device from the VPN should trigger the natural disconnect tracking method

Reply:
I wonder why this had been added. To expedite the state transition for some reason?